ui_framework/src/components/table/listing_table/listing_table_row.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  KuiTableRow,
+  KuiTableRowCheckBoxCell,
+} from '../';
+
+export class KuiListingTableRow extends React.PureComponent {
+  onCheckChanged = () => {
+    this.props.onCheckChanged(this.props.row.id);
+  };
+
+  render() {
+    const { row, isChecked } = this.props;
+    return (
+      <KuiTableRow>
+        <KuiTableRowCheckBoxCell
+          isChecked={isChecked}
+          onChange={this.onCheckChanged}
+        />
+        {row.cells}
+      </KuiTableRow>
+    );
+  }
+}
+
+KuiListingTableRow.PropTypes = {
+  row: PropTypes.shape({
+    id: PropTypes.string,
+    cells: PropTypes.arrayOf(PropTypes.node),
+  }).isRequired,
+  onCheckChanged: PropTypes.func.isRequired,
```

Comment:
Can we change the language in this component from "checked" to "selected", since that's more consistent with the language we use elsewhere and also technically more accurate since this component is a selectable row and not a checkable checkbox?